src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -88,14 +88,20 @@ function initStore() {
 
 
             function getMolecularProfiles(query:any){
-                //if there's only one study, we read profiles from query params and filter out udfine
-                const molecularProfiles = [
+                //if there's only one study, we read profiles from query params and filter out undefined
+                let molecularProfiles = [
                     query.genetic_profile_ids_PROFILE_MUTATION_EXTENDED,
                     query.genetic_profile_ids_PROFILE_COPY_NUMBER_ALTERATION,
                     query.genetic_profile_ids_PROFILE_MRNA_EXPRESSION,
                     query.genetic_profile_ids_PROFILE_PROTEIN_EXPRESSION,
                 ].filter((profile:string|undefined)=>!!profile);
 
+                // append 'genetic_profile_ids' which is sometimes in use
+                molecularProfiles = molecularProfiles.concat(query.genetic_profile_ids || []);
```

Comment:
this comes from studyview?

Reply:
i mean case where param exists.  i actually want it to always except list as the above is super annoying.